<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="processes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Процеси</string>
    </property>
    <widget class="QTextBrowser" name="processesView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>561</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="possibilityOfIncome">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>450</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Імовірність надходження:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Межі довжини (байт)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Мінімальна:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="minimumTaskLength">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>510</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальна:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="maximumTaskLength">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>540</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>421</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Пам'ять</string>
    </property>
    <widget class="QTextBrowser" name="memoryView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="nextTick">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>650</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NextTick</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
